### PR TITLE
Qmaps 721 search field improve

### PR DESCRIPTION
**Pull request by Julien-laville**
- add shadow to the top decoration
- display correct value for address after url load
- add clear field function

clean
 - double field update
 - clear import

### CODE REVIEW

File: public/images/backgrounds/top_desktop.svg [hunk 1]
```diff
@@ -1,14 +1,33 @@
...
-        <path fill="#76BC54" d="M197.718 63c-.475-4.435-.718-8.94-.718-13.5 0-17.578 3.614-34.312 10.14-49.5h230.72C444.387 15.188 448 31.922 448 49.5c0 4.56-.243 9.065-.718 13.5H197.718z"/>
-        <path fill="#4BA2EA" d="M288.603 0a216.031 216.031 0 0 1-21.69 63H71C46.494 56.715 23.028 49.44-.001 42V0h288.604z"/>
-        <path fill="#65CEE7" d="M183.714 0a177.173 177.173 0 0 1-27.819 63H71C46.494 56.715 23.028 49.44-.001 42V0h183.715z"/>
-        <path fill="#87C966" d="M519 0h319c-34.382 24.883-79.182 46.397-130 63-83.84-.014-146.84-.014-189 0V0z"/>
-        <path fill="#87C966" d="M386.542 63c-12.283-18.494-20.505-39.918-23.393-63h292.702a146.867 146.867 0 0 1-3.364 17.853A146.746 146.746 0 0 1 632.458 63H386.542z"/>
-        <path fill="#9DDC7D" d="M472.013 63c-13.805-17.773-22.881-39.403-25.34-63h246.654a123.566 123.566 0 0 1-3.121 17.56c-4.25 16.771-11.919 32.179-22.22 45.44H472.014z"/>
-        <use fill="#76BC54" xlink:href="#a"/>
+    <g id="Elements/Background-search/Desktop/Maps" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
+        <g id="Group" filter="url(#filter-1)" transform="translate(0.998882, 0.000000)">
```
Comment (amatissart) [inline public/images/backgrounds/top_desktop.svg]: this extra "translate" causes a blank border on the left, and should be removed.

File: src/adapters/poi/idunn_poi.js [hunk 2]
```diff
@@ -22,6 +22,15 @@ export default class IdunnPoi extends Poi {
     this.bbox = rawPoi.geometry.bbox
   }
 
+  getInputValue() {
+    switch (this.type) {
+      case 'address':
```
Comment (amatissart) [inline src/adapters/poi/idunn_poi.js]: the same for 'street'

File: src/views/direction/direction.dot [hunk 3]
```diff
@@ -4,16 +4,18 @@
...
+          <button class="icon-x itinerary__field__clear" type="button" {{= click(this.clearOrigin, this) }}></button>
         </div>
       </form>
-      <form onsubmit="fire('submit_direction_destination');return false">
+      <form onsubmit="fire('submit_direction_destination');return false" novalidate>
         <div class="itinerary_field_destination">
           <div class="itinerary_icon itinerary_icon_destination icon-map-pin"></div>
-          <input id="itinerary_input_destination" autocomplete="off" spellcheck="false" value="{{= this.destination ? this.destination.name : '' }}" placeholder="{{= _('End point', 'direction') }}">
+          <input id="itinerary_input_destination" autocomplete="off" spellcheck="false" value="{{= this.destination ? this.destination.getInputValue() : '' }}" required placeholder="{{= _('End point', 'direction') }}">
+          <button class="icon-x itinerary__field__clear" type="button" {{= click(this.clearDestination, this) }}></button>
```
Comment (amatissart) [inline src/views/direction/direction.dot]: these new  "clear" buttons are not visible on mobile

File: public/images/backgrounds/top_desktop.svg (hunk 1 above)
Comment (Julien-laville) [reply public/images/backgrounds/top_desktop.svg]: WOW !!!

File: src/adapters/poi/idunn_poi.js (hunk 2 above)
Comment (Julien-laville) [reply src/adapters/poi/idunn_poi.js]: sure

**Pull request merged**